### DISCUSSION TITLE
这个项目是能自动扫描ip吗？

### DISCUSSION THREAD

**Discussion by name2008less** [SeaHOH/GotoX#44] [问答/咨询] [answered]
首页说明看不太懂 ，貌似是可以和xx一样优选ip？？  还有其他和goproxy的区别么？

**Comment by SeaHOH** [comment, marked as answer]
- 首先，你用扫描工具扫描或其它来源得到一批可用的 IP （`ip.txt` 数量上千为宜）放入 GotoX 配置中；
- 然后，在运行过程中 GotoX 会挑选 10 个左右（可配置）速度较快的 IP 来使用；
- 筛选过程会一直持续，保持使用的 IP 是速度叫快的；
- 还有一种固定模式，不用一直筛选，但是需要质量非常稳定的 IP （`Config.ini` 数量一百以下为宜）才能保证正常使用。

**Reply by SeaHOH** [reply]
现在使用的 CFW 没有筛选 IP，直连谷歌仍然可以使用筛选功能。
